### PR TITLE
Feat: Create Section Contact Us

### PR DESCRIPTION
**Pull request by adzharamrullah**
## Changes

- Feat: Create Section Contact Us

Evidence

title: Feat: Create Section Contact Us
project: Desa Digital
participants: @adzharamrullah

### CODE REVIEW

File: components/Base/Button/Index.vue [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <template>
-  <jds-button v-bind="{...$props,...$attrs}" v-on="$listeners" />
+  <jds-button v-bind="{...$props,...$attrs}" v-on="$listeners">
+    <div class="flex gap-1 items-center justify-center">
+      <slot name="icon" />
+      <slot name="label" />
+    </div>
```
Comment (Ibwedagama) [inline components/Base/Button/Index.vue:6]: love it ❤️

File: components/ContactUs/index.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          </h3>
+        </div>
+        <div class="contact-us__cta">
+          <BaseButton class="contact-us__cta-btn">
+            <template #icon>
+              <img src="~/assets/icons/Whatsapp.svg" alt="Icon Whatsapp">
+            </template>
+            <template #label>
+              Hubungi Tim Kami Via Whatsapp
+            </template>
```
Comment (Ibwedagama) [inline components/ContactUs/index.vue:20]: minor:

maybe we can make `#label` to default slot? @adzharamrullah

**Review by Ibwedagama — APPROVED** [review]

File: components/ContactUs/index.vue (hunk 2 above)
Comment (adzharamrullah) [reply components/ContactUs/index.vue:20]: Maybe 😂

Comment (adzharamrullah) [reply components/ContactUs/index.vue]: here refactored slot label to default slot https://github.com/jabardigitalservice/dedi-landing-page/pull/9/commits/7cdc3b3458cea78247c5a072d87510c7a7efc563

**Pull request merged**